README.md

```diff
@@ -1,14 +1,144 @@
+
+2. Click on _Get started_.
+
+ * You can skip through these next screens by clicking _Next_
+  * Click _Continue to the console_
+
+2. OPTIONAL Add a custom domain.
+
+ * In the hosting dashboard, click _Add custom domain_
+ * Enter the details for the domain to use (eg. `example.sparkle.space`)
+    **Note:** you will also need to configure the DNS to make this work properly (see below), and this may take 24-48hrs to take effect
+ * Don't check 'Redirect `example.sparkle.space` to an existing website'
+ * Click _Continue_
+ * If you haven't verified your domain before, you'll need to do this here. You will likely need to create a DNS 'TXT' record to prove that you own the domain, containing a `google-site-verification` value
+ * You can follow similar steps to the ones described below to do this: _Enter domain --> Verify ownership --> Go live_
+ * You will need to enter the provided DNS 'A' records into your domain registrar (eg. GoDaddy) to point your domain at the Firebase servers (record type, host, value)
+
+You can close this page and return to the Firebase console.
+
+</details>
+
+
+### Step 5: Set up Firebase Authentication
+
+1. From the Firebase console, within the appropriate project, click on ['Authentication'](https://console.firebase.google.com/u/2/project/`example-project`/authentication) on the left hand menu.
+
+2. Click on _Get Started_.
+
+  * In the ‘Sign-in method’ tab:
+
+    * Hover over ‘Email/Password’, then click the edit icon (pencil)
+    * Select ‘enabled’ (but leave email link ‘passwordless sign in’ disabled)
+    * Click _Save_
+    * Scroll down to 'Authorised domains'
+    * Click _Add domain_
+    * In the 'Domain' field, enter the custom domain we created above (eg. `example.sparkle.space`)
+    * Click _Add_
+    * Scroll down to 'Advanced'
+
+
+  3. _Note that there is a 'Manage sign-up quota' setting, which defaults to `100` signups per hour from the same IP address. Under most normal cases, this probably doesn't need to be changed, but sometimes (eg. employees all accessing from the same office, therefore having the same IP address), you may need to temporarily increase this limit._
+
+    * To do this, click _Change_
+        * Configure 'Sign-ups per hour', 'Start Date', 'Time', and 'Duration (days)' as required
+         * 'Sign-ups per hour' must be between `1` and `1000`
+         * 'Start Date' must be in the future
+         * 'Duration (days)' must be between `1` and `7`
+    * Click _Save_
+    * Notice the 'Sign-up quota change scheduled' notification
+
+This part of the setup is complete!
+
+
+### Step 6: Additional Notes
+
+If you're interested in setting up the project to run in your own environment but you're not sure how to go about it, feel free to [open an issue](https://github.com/sparkletown/sparkle/issues/new) asking for assistance.
```

Comment:
This line basically existed as a catchall before we had public documentation for this. Not sure if it's going to be relevant anymore once we do.